app/code/Magento/Checkout/Model/PaymentInformationManagement.php

```diff
@@ -85,12 +94,15 @@ public function savePaymentInformationAndPlaceOrder(
         try {
             $orderId = $this->cartManagement->placeOrder($cartId);
         } catch (\Magento\Framework\Exception\LocalizedException $e) {
+            /** @var \Magento\Quote\Model\Quote $quote */
```

Comment:
Please get rid of this unnecessary doc block. We have return type at doc block for "cartRepository->getActive" method. It's more than enough for this method.